src/core/server/graph/tenant/schema/schema.graphql

```diff
@@ -2761,6 +2761,12 @@ input CreateCommentInput {
   """
   body: String!
 
+  """
+  commentOrderBy is how the comments are ordered when the
+  comment was created.
+  """
+  commentsOrderBy: String
+
```

Comment:
This should not be passed to the server, this is purely a client side parameter.